Automattic/jetpack
# My Jetpack: Update Jetpack AI description, features and image

Fixes #30553

## Proposed changes:

* Updates the image with the one from figma file gG1smMHbNGas0qwP9y6RwK-fi-3660_12984
* Updates the description and features with the ones from the figma file 

**Note**: I left out `images` from the feature list proposed in the figma file

![image](https://github.com/Automattic/jetpack/assets/746152/deb552c3-cde0-4e71-b2e8-a501ec6d744a)

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?
NO

## Testing instructions:
* Visit `p-admin/admin.php?page=my-jetpack#/add-jetpack-ai`
* Confirm the interstitial looks as in the image above
